app/controllers/posts_controller.rb

```diff
@@ -33,13 +31,25 @@ def destroy
 		render json: { message: 'post deleted successfully.' }
 	end
 
+	def users_post
+		current_user_posts = current_user.posts
+		render json: { post: current_user_posts }, status: :ok
+	end
+	
+	# def users_post
+	# 	# current_user_posts = current_user.posts
+	# 	user = User.find(id: params[:id])
+	# 	user_post = user.posts
+	# 	render json: { post: user_post }, status: :ok
+	# end
+
 	private
 
 	def set_post
 		@post = Post.find(params[:id])
 	end
 
 	def post_params
-		params.require(:post).permit(:name, :description, :status)
+		params.require(:post).permit(:name, :description, :status, :rejected_reason)
```

Comment:
What is use of rejected reason here on post creation, this will be done on active admin?